enhancements/authentication/pod-security-admission.md

```diff
@@ -0,0 +1,113 @@
+---
+title: pod-security-admission
+authors:
+  - "@s-urbaniak"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-09-14
+last-updated: 2021-09-14
+status: informational
+see-also:
+replaces:
+superseded-by:
+---
+
+# PodSecurity admission in OpenShift
+
+Kubernetes recently gained PodSecurity policy admission as part of KEP 2579 [1].
+It is a new pod security admission plugin to enforce Kubernetes PodSecurity standards [2].
+It is meant to replace the existing PodSecurityPolicy admission mechanism in Kubernetes.
+
+This enhancement describes the migration path to integrate PodSecurity admission inside OpenShift.
+Special interest is put to design a migration scheme which lets PodSecurity coexist with the existing Security Context Constraints (SCC) mechanism.
+
+[1] https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2579-psp-replacement
+[2] https://kubernetes.io/docs/concepts/security/pod-security-standards/
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Motivation
+
+In contrast to PodSecurityPolicy admission the desire is not to disable the new upstream PodSecurity admission plugin subsystem within OpenShift.
```

Comment:
i'm not clear on what the "in contrast to" is contrasting.

Reply:
`PodSecurityPolicy` is disabled in OpenShift but there is a desire to enable the `PodSecurity` admission. Do you have a suggestion how I can phrase this better (sorry, non-native speaker) :grimacing:

Reply:
"Unlike the (now deprecated?) PSP admission plugin which is disabled in openshift, the goal here is to enable the PS admission plugin within openshift"

Reply:
This sounds much better, thank you! I will add this sentence instead :+1:

Reply:
clarification: podsecurity is indeed deprecated as of 1.21 (see https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/)

Reply:
I will add this reference.